tests/code/test_events.py

```diff
@@ -896,3 +897,48 @@ def test_getEventRsvpCount():
             assert rsvpd_user_count == (index + 1)
 
         transaction.rollback()
+
+
+@pytest.mark.integration
+def test_copyRsvpToNewEvent():
+    with mainDB.atomic() as transaction:
+        with app.app_context():
+            g.current_user = "heggens"
```

Comment:
Why do we set g.current_user. Doesn't seem to be used anywhere else?

Reply:
It's used by the createRsvpLog function called in copyRsvpToNewEvent.